### DISCUSSION TITLE
Sunbird workspace not working , session will end with timeout in localhost:3000

### DISCUSSION THREAD

**Discussion by santhosh-tarento** [Sunbird-Ed/Community#579] [Installation] [answered]
create course, create question set, profile view session gets end in http://localhost:3000 with 401 request

**Comment by rajeevsathish** [comment]
@santhosh-tarento there are some routes which do not work from the local to point to the upstream as there is some session problem team is working on this will update in the thread once the issue is fixed.

cc: @vinukumar-vs

**Reply by chitranshu-keshav** [reply]
@rajeevsathish Can you please mention the JIRA ticket related to this issue

**Reply by rajeevsathish** [reply, marked as answer]
@santhosh-tarento  and @chitranshu-keshav this is the ticket for this issue

https://project-sunbird.atlassian.net/browse/ED-2616
